forecasttools/trajectories_to_quantiles.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import polars as pl
+
+
+def trajectories_to_quantiles(
+    trajectories: pl.DataFrame,
+    quantiles: list[int] = None,
+    timepoint_cols: list[str] = ["timepoint"],
+    value_col: str = "value",
+    quantile_value_name: str = "quantile_value",
+    quantile_level_name: str = "quantile_level",
+    id_cols: list[str] = None,
+) -> pl.DataFrame:
+    # set default quantiles
+    if quantiles is None:
+        quantiles = (
+            [0.01, 0.025] + list(np.arange(0.05, 1.0, 0.05)) + [0.975, 0.99]
+        )
+    # group trajectories based on timepoint_cols and id_cols
+    group_cols = (
+        timepoint_cols if id_cols is None else timepoint_cols + id_cols
+    )
+
+    # compute quantiles
+    def compute_quantiles(values):
+        quantile_values = np.quantile(values, quantiles)
+        return quantile_values
+
+    # get quantiles across epiweek for forecast
+    quantile_results = trajectories.group_by(group_cols).agg(
+        [
+            pl.col(value_col)
+            .map_elements(
+                lambda group_values: compute_quantiles(group_values),
```

Comment:
Since you're already defining a lambda function, just use `np.quantile` here instead of separately defining `compute_quantiles`

Reply:
Sounds good. I originally had `compute_quantiles` return a dictionary (with `quantile_levels` and `quantile_values`) and `map_elements` receive a `pl.Object` but this was more complicated than it needed to be.